Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM ghcr.io/cloudflare/pint:0.63.0
+FROM ghcr.io/cloudflare/pint:0.64.0
```

Comment:
I think it's worth to bump action version to v2, since 0.64.0 brings breaking changes to config.

https://github.com/cloudflare/pint/releases/tag/v0.64.0